.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: CI
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  job1:
+    name: Nim Tests
+    runs-on: ubuntu-18.04
+    container: nimlang/nim:1.2.4-ubuntu-regular@sha256:02a555518a05c354ccb2627940f6138fca1090d198e5a0eb60936c03a4875c69
+
+    steps:
+    - uses: actions/checkout@aabbfeb2ce60b5bd82389903509092c4648a9713 # v2.2.0
```

```suggestion
    - uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b # v2.3.1
```

(Non-blocking suggestion).

I know that the `ci.yml` here is a copy of the [ci.yml](https://github.com/exercism/nim-test-runner/blob/0870db148b679665697099de5c01ad42e632839b/.github/workflows/ci.yml) in `exercism/nim-test-runner`, but I think we might as well use the latest release when creating this file.

It looks like the bump includes a change that might be relevant (under some conditions) if/when the default branch is renamed from `master`,